syntax = "proto3";

package notification;

service EmailService {
  rpc SendEmail (EmailRequest) returns (EmailResponse);
}

message EmailRequest {
  string recipient_email = 1;
  string subject = 2;
  string message = 3;
  string sender_email = 4;
}

message EmailResponse {
  bool success = 1;
  string message = 2;
}
